sys-devel/fpcupdeluxe/additional-files/install_fpcupdeluxe.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+installDir="$(finddir B_USER_NONPACKAGED_DIRECTORY)/bin/"
+binaryDir="$(finddir B_SYSTEM_BIN_DIRECTORY)"
+mkdir -p $installDir
+
+cp -f "$binaryDir/fpcupdeluxe" "$installDir"
```

Comment:
@korli is there a way to "move" this from $binaryDir? Launching it from Terminal (will use $prefix/bin/fpcupdeluxe) won't work. Tried with `mv` instead of `cp -f`, but that doesn't actually move the binary.